### ISSUE TITLE
Add role="modal" to the modal dialog component

### ISSUE THREAD

**Issue by Veyfeyken** [digipolisantwerp/acpaas-ui_react#208]
## Problem

The react-modal was definitely build with accessibility in mind. Only 2 issues:

* the parent div lacks a role="modal" attribute
* the close button doesn't have an accessible name, it's empty

## Suggestion

* Add role="modal" to the parent div
```html
<div class="ReactModal__Overlay ReactModal__Overlay--after-open m-overlay is-active" aria-modal="true">
```
* Add "sluiten" to the close button, as an aria-label or hidden text.
```html
<button class="m-modal__close a-button--default has-icon a-button-transparent" title="" alt="" aria-label="">
<span class="fa fa-times"></span>
</button>
```

## Articles and demo
* [ARIA design pattern for modal dialog](https://www.w3.org/TR/wai-aria-practices-1.2/#dialog_modal)
* [Example of modal dialog](https://www.w3.org/TR/wai-aria-practices-1.2/examples/dialog-modal/dialog.html)
